tsconfig.esm.json

```diff
@@ -3,17 +3,12 @@
   "compilerOptions": {
     "module": "es2020",
     "outDir": "./dist/esm",
```

Comment:
In the old we had the output on `./dist` for the esm module.

That allows you to import `import { Query } from '@aboutbits/react-pagination/dist/engine'`.

I'm not sure if the current version would work because you tell them to always look into `/dist/esm`

Reply:
I chose this output directory so that the user of the library knows what is being used. Because we have no entry point, the user must choose. I think that naming the directories `dist/esm` and `dist/cjs` is more clear than naming them `dist` and `dist/cjs` or `dist/esm` and `dist`.
Do you think it is better to default to `esm` or `cjs`? If yes, which one?

Reply:
In the old package `@aboutbits/react-pagination` and the package `@aboutbits/react-toolbox` we use `cjs` as the default. What default should I take?

Reply:
I see that the build of Next.js is failing when importing the `esm` version. So I would default to `cjs`

Reply:
Thanks for checking. Go ahead with `cjs.